client-session/src/main/java/com/iluwatar/client/session/Request.java

```diff
@@ -31,12 +31,12 @@
 /**
  * The Request class which contains the Session details and data.
  */
-@Data
-@AllArgsConstructor
-public class Request {
-
-  private String data;
-
-  private Session session;
+//@Data
+//@AllArgsConstructor
+public record Request (String data, Session session) {
+//
+//  private String data;
+//
+//  private Session session;
```

Comment:
The commented out parts should be removed

Reply:
Check this elsewhere too

Reply:
do you want me to even remove builder annotation usage.